build/devops_gated.yml

```diff
@@ -122,7 +122,7 @@ jobs:
 - job: linuxubuntu
   displayName: 'Build Linux Ubuntu'
   pool:
-    name: Azure-MessagingStore-LinuxBuildMachinePool
+    name: Azure-MsgStore-Linux2204BuildMachinePool
```

Comment:
>me: Azure-MsgStore-Linux2204BuildMachinePool [](http://example.com/codeflow?start=6&length=44)

Hmm, is this intended?

Reply:
This seems to be from another change? Or is it needed for this test to pass?

Reply:
The test fails more on the new version of linux, new helgrind I'm guessing.